patches/minecraft/net/minecraft/entity/EntityCreature.java.patch

```diff
@@ -0,0 +1,30 @@
+--- ../src-base/minecraft/net/minecraft/entity/EntityCreature.java
++++ ../src-work/minecraft/net/minecraft/entity/EntityCreature.java
+@@ -15,6 +15,9 @@
+     private BlockPos field_70775_bC = BlockPos.field_177992_a;
+     private float field_70772_bD = -1.0F;
+     private final float field_184661_bw = PathNodeType.WATER.func_186289_a();
++    protected float maxLeashDistance = 10.0F; // forge: make maxLeashDistance a field instead of hard-coded
++    public float getMaxLeashDistance() { return maxLeashDistance; }
++    public void setMaxLeashDistance(float distance) { maxLeashDistance = distance; }
```

Comment:
Forge added methods are at the bottom of the class, and you need to add javadocs explaining what this field is for.